db/models/applications.js

```diff
@@ -97,6 +100,23 @@ class PendingApplication extends Model {
             order: [['createdAt', 'DESC']]
         });
     }
+    static findExpiredApplications() {
+        return this.findAll({
+            attributes: ['id'],
+            where: {
+                expiresAt: {
+                    [Op.lte]: moment().toDate()
```

Comment:
Interesting! I thought Sequelize 5.x and above, we couldn't use `lte` like operators on dates.

Reply:
It's working for me! Where did you see that?

Reply:
I don't have the exact link but I was sure I read it on StackOverflow. Anyways as long as its working, its fine I guess!